src/adapters/sqlite/encodeQuery/test.js

```diff
@@ -270,4 +270,14 @@ describe('SQLite encodeQuery', () => {
     expect(() => encoded([Q.unsafeLokiExpr({ hi: true })])).toThrow('Unknown clause')
     expect(() => encoded([Q.unsafeLokiTransform(() => {})])).toThrow('not supported')
   })
+  it('encodes JOIN over FTS table', () => {
```

Comment:
pls place this in a more appropriate spot, say :127 . I'd also suggest adding a complex test (or modifying a complex test such as `encodes JOIN queries`) to verify that Q.ftsMatch composes nicely with Q.on, nesting, etc.